pkg/http/server_test.go

```diff
@@ -97,3 +97,66 @@ func TestNewServer(t *testing.T) {
 		}
 	}
 }
+
+var tlsCipherSuitesTests = []struct {
```

Comment:
1. move this testcases inside the test

Reply:
Makes reading the function itself harder - it's common in Go to separate test function and test vectors.

Reply:
This is the style we use across all our code base and don't see any problem of readability so far.

Reply:
Here it does not make a big difference but as soon as the number of test vectors grows the functions become very large - most of the function consists of test vectors. That's why the Go standard library focus the table driven test style used here - See:
https://github.com/golang/go/blob/master/src/regexp/find_test.go#L28
https://github.com/golang/go/blob/master/src/crypto/aes/aes_test.go#L134
https://github.com/golang/go/blob/master/src/html/escape_test.go#L21

But I can change it if fix the"function-centric" as the style used by minio.